packages/hooks/launchpad/useCompleteCollection.ts

```diff
@@ -113,7 +114,7 @@ export const useCompleteCollection = () => {
             );
 
             metadatas.push({
-              image: image.hash,
+              image: CID.parse(image.url).toV0().toString(),
```

Comment:
you didn't read what I said, this should be an ipfs uri, not a raw cid

and I don't understand why you need to use a v0 cid

please see https://docs.opensea.io/docs/metadata-standards#ipfs-and-arweave-uris